### PR TITLE
Create Synthetic Seed Data for MPI

### LINKED ISSUE #414: Create testing data based on MPI seed data from LAC
### Why are we doing this?

The next phase of work to translate the seed Parquet files into MPI records will be easier if there is synthetic (non-PHI) data to work with, so that local development can be done instead of running everything in the LAC Azure environment.

### Action Requested

* Look at the MPI seed data that LAC added as a Parquet file to their Azure instance on 4/7.
* Using the same file structure and schema, create a synthetic data set.

### Acceptance Criteria

A synthetic data set is available for engineers working on MPI seeding to use.

### Additional Context

This work will need to be a collaboration between the cloud team and record linkage engineers.

### PR DESCRIPTION
**Pull request by m-goggins**
## Summary
Quick script to generate a parquet file intended to mimic the patient data that will be used to seed the LAC MPI. The synthetic data include the following columns:

- 'iris_id'
- 'birthdate': YYYY-MM-DD
- 'ssn':  All values are "123-456-7890" since we do not use it in our algorithm
- 'mrn': unique
- 'first_name'
- 'middle_name': all "Middle" since we do not use it in our algorithm 
- 'last_name'
- 'sex': M/F only because of Synthea limitations
- 'address'
- 'city'
- 'state',
- 'zip'
- 'email'
- 'home_phone' 
- 'cell_phone'

## Related Issue
Fixes #414

## Additional Information
Anything else the review team should know?

[//]: # (PR title: Remember to name your PR descriptively!)

### CODE REVIEW

**Comment by m-goggins** [comment]
Putting this one up for review since we haven't heard back from LAC

File: examples/MPI-sample-data/build_synthetic_mpi_parquet.py [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+from random import randint
+from string import ascii_letters
+
+random.seed(410)
+
+df = pd.read_csv(
+    # locally stored file of synthetic LAC synthea data; user must change
+    "~/20230106_LAC_10000_123_456/csv/patients.csv",
+    usecols=[
+        "Id",
```
Comment (emmastephenson) [inline examples/MPI-sample-data/build_synthetic_mpi_parquet.py:14]: Should this be ID for consistency?

File: examples/MPI-sample-data/build_synthetic_mpi_parquet.py [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+    """
+    Adds an "email" column with a synthetic email address for each row of data. The
+    email address domains are limited to the most common providers in the US (gmail,
+    yahoo, and hotmail).
+
+    :param data: A DataFrame object.
+    :return: A DataFrame object with an EMAIL column of synthetic email addresses.
+    """
+    emails = [
+        "".join(random.choice(ascii_letters) for x in range(10))
```
Comment (emmastephenson) [inline examples/MPI-sample-data/build_synthetic_mpi_parquet.py:39]: I'm curious if you've looked into the `faker` library at all? We used it on a previous project, and it had a couple helpful utilities like `faker.email()` or `faker.phone_number()`.

**Review by emmastephenson — APPROVED** [review]
LGTM, thanks for tackling this! Should be really helpful to have it across the team.

File: examples/MPI-sample-data/build_synthetic_mpi_parquet.py (hunk 1 above)
Comment (m-goggins) [reply examples/MPI-sample-data/build_synthetic_mpi_parquet.py:14]: This file is automatically generate by Synthea so I'm hesitant to change the headers, but yeah I don't know why they do it like that!

Comment (emmastephenson) [reply examples/MPI-sample-data/build_synthetic_mpi_parquet.py:14]: Ahh gotcha! In that case leaving as is works 👍

**Pull request merged**